install/operator/pkg/generator/assets/install/addon-legacyui.yml

```diff
@@ -0,0 +1,355 @@
+{{if eq (or .Syndesis.Spec.Addons.legacyui.enabled "false") "true"}}
```

Comment:
I think we can drop the legacy UI entirely, no need to drag that along.

Reply:
Don't mind dropping legacy!  https://github.com/syndesisio/syndesis/issues/6263